README.md

```diff
@@ -65,23 +66,23 @@ Place brief descriptions of Pipelines here
 
 - Deploy to Gov PaaS - This is a simple pipeline to demonstrate capabilities.  Builds, tests and deploys a simple python application to the PaaS for evaluation in Dev and Test Only.
 
-Testing
+# Testing
 
 Unit & Accessibility Testing
 To run all tests including aXe accessibility tests (using Chrome driver for Selenium) in a development environment run:
 
 ...on macOS
 
-pytest --driver Chrome --driver-path .venv/lib/python3.10/site-packages/chromedriver_py/chromedriver_mac64
+    pytest --driver Chrome --driver-path .venv/lib/python3.10/site-packages/chromedriver_py/chromedriver_mac64
 ...on linux64
 
-pytest --driver Chrome --driver-path .venv/lib/python3.10/site-packages/chromedriver_py/chromedriver_linux64
+    pytest --driver Chrome --driver-path .venv/lib/python3.10/site-packages/chromedriver_py/chromedriver_linux64
 ...on win32
 
-pytest --driver Chrome --driver-path .venv/lib/python3.10/site-packages/chromedriver_py/chromedriver_win32.exe
+    pytest --driver Chrome --driver-path .venv/lib/python3.10/site-packages/chromedriver_py/chromedriver_win32.exe
 The aXe reports are printed at /axe_reports
 
-This repo comes with a .pre-commit-config.yaml, if you wish to use this do
+This repo comes with a `.pre-commit-config.yaml`, if you wish to use this do
```

Comment:
🔧 I think if we change out chromedriver-py to web driver manager then this is not needed, see the discovery frontend for an example of this.

Reply:
I haven't changed any of the chromedriver stuff in this PR, I just got carried away formatting the readme ;-) But thinking about it... the notifications service doesn't have a front end, does it even need the chromedriver testing stuff?

Reply:
Yeah doing a find in files the only references to the chromedriver are in readme and requirements.txt so I will remove them!